src/commands/horde.ts

```diff
@@ -23,11 +23,11 @@ export default class extends Command {
             color: Colors.Blue,
             title: "AI Horde",
             //TODO: Add more info in the future
-            description: `The [AI Horde](https://https://aihorde.net/) is a crowdsourced service providing Stable Diffusion and other AI services for everyone.\nIt is free, open sourced, and relies on volunteer processing power.\n\nIf you enjoy using it, please consider [onboarding your own PC as a worker](https://github.com/db0/AI-Horde-Worker#readme) or supporting its development [on patreon](https://www.patreon.com/db0)${article ? `\n\n**Latest News**\n${article.newspiece}\n<t:${Math.round(Number(new Date(article.date_published!))/1000)}>` : ""}`
+            description: `The [AI Horde](https://https://aihorde.net/) is a crowdsourced service providing Stable Diffusion and other AI services for everyone.\nIt is free, open sourced, and relies on volunteer processing power.\n\nIf you enjoy using it, please consider [onboarding your own PC as a worker](https://github.com/AI-Horde/AI-Horde-Worker#readme) or supporting its development [on patreon](https://www.patreon.com/db0)${article ? `\n\n**Latest News**\n${article.newspiece}\n<t:${Math.round(Number(new Date(article.date_published!))/1000)}>` : ""}`
```

Comment:
please make sure that the link is actually valid, in this case it is not valid and returns a 404